raiden_contracts/tests/deprecation_switch_testnet.py

```diff
@@ -65,8 +65,8 @@ def deprecation_test(
 
     # Check that we can open channels and deposit on behalf of A and B
     # Some arbitrary Ethereum addresses
-    A = "0x6AA63296FA94975017244769F00F0c64DB7d7115"
-    B = "0xc9a4fad99B6d7D3e48D18d2585470cd8f27FA61e"
+    A = HexAddress(HexStr("0x6AA63296FA94975017244769F00F0c64DB7d7115"))
```

Comment:
Why do we need the `HexStr`? Can't we cast to `HexAddress` directly?

Reply:
Yeah, that one is weird:
```
mypy raiden_contracts/ setup.py
raiden_contracts/tests/deprecation_switch_testnet.py:68: error: Argument 1 to "HexAddress" has incompatible type "str"; expected "HexStr"

Reply:
It's really defined that way:
```
HexAddress = NewType('HexAddress', HexStr)
```
see https://github.com/ethereum/eth-typing/blob/965459459f1a4f1090ccc004b433c11fb4ad97b4/eth_typing/evm.py#L16

We've never cascaded NewTypes in raiden, so this was unusual for me.